src/main/java/com/epam/training/auction/repository/UserRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.epam.training.auction.repository;
+
+import com.epam.training.auction.entity.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface UserRepository extends JpaRepository<User, Long> {
+    User findByUserName(String userName);
+}
```

Comment:
You can try to use here name like findByName JPA smart enough to understand this